### PR TITLE
Fall back to IPv4 in devfs_test.dart

### PR DESCRIPTION
**Pull request by tvolkert**
Fixes the broken build in the Chromium bots

### CODE REVIEW

**Comment by Hixie** [comment]
FWIW, always feel free to land fixes like this TBR=ianh if the tree is red and you're fixing the tree.

LGTM.

File: packages/flutter_tools/test/devfs_test.dart [hunk 1]
```diff
@@ -359,8 +359,14 @@ class MockVMService extends BasicMock implements VMService {
   VM get vm => _vm;
 
   Future<Null> setUp() async {
-    _server = await HttpServer.bind(InternetAddress.LOOPBACK_IP_V6, 0);
-    _httpAddress = Uri.parse('http://[::1]:${_server.port}');
+    try {
+      _server = await HttpServer.bind(InternetAddress.LOOPBACK_IP_V6, 0);
+      _httpAddress = Uri.parse('http://[::1]:${_server.port}');
+    } on SocketException {
```
Comment (yjbanov) [inline packages/flutter_tools/test/devfs_test.dart]: SocketException could happen for other reasons. Is there a way to narrow it down more?

**Review by yjbanov — APPROVED** [review]

File: packages/flutter_tools/test/devfs_test.dart (hunk 1 above)
Comment (tvolkert) [inline packages/flutter_tools/test/devfs_test.dart]: I could check the errno on the associated `OSError`, but I think that may be overkill?

**Review by cbracken — APPROVED** [review]

File: packages/flutter_tools/test/devfs_test.dart (hunk 1 above)
Comment (cbracken) [inline packages/flutter_tools/test/devfs_test.dart]: `OSError` values errno etc. values differ across platforms. Feels like a bit of a rat's nest to deal with them if we can avoid it. For a test this seems fine.

**Pull request merged**